src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -196,4 +200,154 @@ Scalar<DataType> time_deriv_of_lapse(
     const tnsr::A<DataType, SpatialDim, Frame>& spacetime_unit_normal,
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
     const tnsr::aa<DataType, SpatialDim, Frame>& pi) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes spatial derivatives of the shift vector from
+ *        the generalized harmonic and geometric variables
+ *
+ * \details Let the generalized harmonic spatial derivative variable be
+ * \f$\Phi_{iab} = \partial_i \psi_{ab} \f$. The spatial derivatives of the
+ * shift vector \f$N^i\f$ are:
+ * \f{align*}
+ *  \psi^{ja} N t^b \Phi_{iab} - 2 t^j \partial_i N
```

Comment:
The prose preceding this line should say something along the lines of "the shift vector \f$N^j\f$ can be derived from the following steps:"

The steps would also be easier to follow if you started with the $\partial_i N^j$ and proceeded to work out the expansion, largely in reverse order of how it's written below.

To the best we were able to tell, the result from the first equation block is not used again, so in the reversed order, perhaps it can just be omitted?

Reply:
Fixed. Reversed order of derivation for various functions.